packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -896,6 +896,7 @@ class ListTile extends StatelessWidget {
       autofocus: autofocus,
       enableFeedback: enableFeedback ?? tileTheme.enableFeedback ?? true,
       child: Semantics(
+        button: onTap != null || onLongPress != null,
```

Comment:
does this need to take enabled into account?

Reply:
if enable is false and onTap is not null, I think it's still a button just a disabled button